src/status_im2/contexts/activity_center/view.cljs

```diff
@@ -85,15 +101,25 @@
 (defn header
   []
   [rn/view
-   [quo/button
-    {:icon                true
-     :type                :blur-bg
-     :size                32
-     :accessibility-label :close-activity-center
-     :override-theme      :dark
-     :style               style/header-button
-     :on-press            #(rf/dispatch [:hide-popover])}
-    :i/close]
+   [rn/view {:style style/header-container}
+    [quo/button
+     {:icon                true
+      :type                :blur-bg
+      :size                32
+      :accessibility-label :close-activity-center
+      :override-theme      :dark
+      :on-press            #(rf/dispatch [:hide-popover])}
+     :i/close]
+    [quo/button
+     {:icon                true
+      :type                :blur-bg
+      :size                32
+      :accessibility-label :activity-center-open-more
+      :override-theme      :dark
+      :on-press            #(rf/dispatch [:bottom-sheet/show-sheet
+                                          {:content        render-more
```

Comment:
I'm a bit confused what `render-more` is? 🤔 
I would have assumed it was some form of pagination function but it seems to be content in it's own bottom sheet. Is there a better way to describe what this use case is?

Reply:
Yes, It's content for the bottom sheet.

I agree that the naming (`render-more`) is misleading. I will change it to something more meaningful. 👍